runtime/build.gradle.kts

```diff
@@ -61,7 +61,7 @@ subprojects {
                 }
             }
 
-            named("jvmTest") {
+            findByName("jvmTest")?.run {
```

Comment:
note: this change was necessary because the new `logging-crt` module does not have a JVM source set, causing errors like "jvmTest does not exist"